models/user.js

```diff
@@ -1,27 +1,33 @@
-'use strict';
-const {
-  Model
-} = require('sequelize');
-module.exports = (sequelize, DataTypes) => {
-  class User extends Model {
-    /**
-     * Helper method for defining associations.
-     * This method is not a part of Sequelize lifecycle.
-     * The `models/index` file will call this method automatically.
-     */
-    static associate(models) {
-      // define association here
-    }
-  };
-  User.init({
-    nickname: DataTypes.STRING,
-    avatarUrl: DataTypes.STRING,
-    email: DataTypes.STRING,
-    pointsForNotGiving: DataTypes.INTEGER,
-    meetingsCount: DataTypes.INTEGER
-  }, {
-    sequelize,
-    modelName: 'User',
-  });
-  return User;
-};
\ No newline at end of file
+const Sequelize = require('sequelize');
+
+
+const users = 'users';
+
```

Comment:
Now as we have configured the database, for the model to be on top of it we have to use it's instance, you can import it like 
```js
const db = require('../../models/index');
```

And then build the model on top of it like so:
```js
const User = db.sequelize.define(
```

All the rest has been done nicely 🥇

Reply:
done